endaq/device/schemata/command-response.xml

```diff
@@ -109,8 +109,6 @@
                 2: CONNECTED, error fields will all be 0, RSSI tag is present in this data structure and represents the strength of the connected net</UIntegerElement>
             <UIntegerElement name="WiFiConnectionError" id="0x5304" multiple="0" mandatory="0">Integer enum. Holds ESP32 error codes, found in https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/wifi.html#wi-fi-reason-code</UIntegerElement>
             <IntegerElement name="RSSI" id="0x5205" multiple="0" mandatory="0">Signal Strength, 0-100, where 100 is perfect. Only present if CONNECTED</IntegerElement>
-            <BinaryElement name="IPV4Address" id="0x5401" multiple="0" mandatory="0"> Network IP address; absent if not connected. </BinaryElement>
-            <BinaryElement name="MACAddress" id="0x5402" multiple="0" mandatory="1"> Network MAC address </BinaryElement>
```

Comment:
I believe these elements are included in the `QueryWiFiResponse`, don't child elements need to be present under the master element?

Reply:
Are you sure? From Teams yesterday:
> [Yesterday 2:59 PM] David Stokes: Does the implemented version of <QueryWiFiResponse> not contain the IPV4 and MAC? [...]
> [Yesterday 3:42 PM] Peter Scheidler: Looks like it just has SSID, ConnectionStatus, ConnectionError

I just wanted to make sure the schema didn't cause additional confusion. I can roll back the changes if there are any concerns.